site/assets/apexcharts.css

```diff
@@ -0,0 +1,679 @@
+@keyframes opaque {
```

Comment:
Maybe add a short comment at the top saying where this CSS file comes from, and how to update it.